### DISCUSSION TITLE
Has anyone had success with BNT006KTL

### DISCUSSION THREAD

**Discussion by LukMaverick** [davidrapan/ha-solarman#254] [Q&A]
Has anyone managed to integrate an inverter before BNT006KTL. Even if I select afore_BNTxxxKTL-2mppt.yaml it shows me nonsense.
![image](https://github.com/user-attachments/assets/ed82e91d-b442-4604-98ea-0e3ac950a315)

**Comment by davidrapan** [comment]
Hey @LukMaverick, did you also tried `afore_2mppt.yaml`?
